data/modules/alpha.yml

```diff
@@ -22,3 +22,13 @@ alpha:
     type: youtube_page
   1-2-4:
     type: formative_assessment
```

Comment:
is this so you have one question per page?  I'm not sure this is needed to be spelled out in the data modules like this, although perhaps given the short timeframe perhaps the only way to do it now.